lib/js/components/Banner/Banner.vue

```diff
@@ -169,20 +221,25 @@
 	&__header {
 		display: flex;
 		justify-content: space-between;
-		padding: 0 $space-4xs;
+		padding: 0 $space-2xs;
 		width: 100%;
 	}
 
 	&__title {
 		@include heading-s-default-bold;
 
-		color: $color-neutral-text-strong;
+		align-items: center;
```

Comment:
does it change anything while `__iconWrapperSmall` has `align-self: stretch;`?

Reply:
Yes, the position of the title text is different when I remove it